components/infobox/wikis/starcraft2/infobox_league_custom.lua

```diff
@@ -614,7 +614,12 @@ function CustomLeague:addToLpdb(lpdbData)
 	end
 	lpdbData.participantsnumber = participantsNumber
 	lpdbData.next = mw.ext.TeamLiquidIntegration.resolve_redirect(CustomLeague:_getPageNameFromChronology(_next))
-	lpdbData.previous = mw.ext.TeamLiquidIntegration.resolve_redirect(CustomLeague:_getPageNameFromChronology(_previous))
+	lpdbData.previous = mw.ext.TeamLiquidIntegration.resolve_redirect(CustomLeague:_getPageNameFromChronology(_previous)) 
+	lpdbData.publishertier = Variables.varDefault('featured')
+	lpdbData.extradata = {
+		seriesnumber = Variables.varDefault('tournament_series_number'),
+		featured = Variables.varDefault('featured')
```

Comment:
Why featured twice?

Reply:
want to switch using `publishertier` in lpdb instead of the featured stuff, but for legacy reasons we need to keep it in extradata for now too